src/ripple/app/main/Application.cpp

```diff
@@ -937,8 +940,15 @@ class ApplicationImp
             std::chrono::seconds {config_->getSize (siSweepInterval)});
     }
 
+    LedgerIndex getMaxLedger() override
```

Comment:
Since this is only the maximum ledger persisted during the last run of the program and is not updated once we start running, consider renaming to `getMinAllowedLedger`.